config/dynamicconfig/development_es.yaml

```diff
@@ -19,8 +19,8 @@ frontend.validSearchAttributes:
       CustomStringField: 0
       CustomKeywordField: 1
       CustomIntField: 2
-      CustomBoolField: 3
-      CustomDoubleField: 4
+      CustomDoubleField: 3
+      CustomBoolField: 4
```

Comment:
For some reason this was misaligned with our proto file.

Reply:
Bleh, maybe we should auto gen these in future. Looks like this will get changed with enums default value changes as well.

Reply:
Yes, this correct, I shouldn't forget about it.